common/src/main/java/com/wynntils/utils/render/type/HealthTexture.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.utils.render.type;
+
+public enum HealthTexture {
```

Comment:
All 3 of these were moved out of CustomBarsOverlayFeature. I am not fully sure if this is the right place for them, but hopefully it is close :)

Reply:
It is close, I agree :) We have some kind of Texture/Resource file somewhere around here as well. I'm not fully sure how we really should handle this, but the current approach is almost good enough, so it can probably stay that way for a while, even if it is not perfect.